addons/binding/org.openhab.binding.gardena/ESH-INF/thing/thing-types.xml

```diff
@@ -828,4 +828,58 @@
 		</state>
 	</channel-type>
 
+	<!-- ############################### Power ############################### -->
+	<thing-type id="power">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account" />
+		</supported-bridge-type-refs>
+
+		<label>Gardena Smart Power plug</label>
+		<description>A Gardena Smart Power plug</description>
+
+		<channel-groups>
+			<channel-group id="device_info" typeId="deviceInfo" />
+			<channel-group id="radio" typeId="radio" />
+			<channel-group id="power" typeId="power" />
+			<channel-group id="firmware" typeId="firmware" />
+		</channel-groups>
+		<properties>
+			<!-- device info -->
+			<property name="manufacturer" />
+			<property name="product" />
+			<property name="serialNumber" />
+			<property name="sgtin" />
+			<property name="version" />
+			<property name="category" />
+		</properties>
+	</thing-type>
+
+	<channel-group-type id="power">
+		<label>Power</label>
+		<description>Power</description>
+		<channels>
+			<channel id="power_timer" typeId="powerTimer" />
+			<channel id="error" typeId="powerError" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="powerTimer">
+		<item-type>String</item-type>
+		<label>Power Timer</label>
+		<description>The power timer</description>
```

Comment:
From this description, it isn't clear to me what it is about and especially, what it reports as a state and which commands I can send. From the example in the readme, I see that "on", "off" and numbers are valid strings - but what do those mean? Is it possible to define state options?